docs/application/web/guides/connectivity/nfc.md

```diff
@@ -492,17 +487,17 @@ When an application no longer needs to receive card emulation events, change the
 
 ## NFC Application Control Operations
 
-You can launch NFC applications based on the NDEF message content using the [application control](./app-management/app-controls-w.md) functionalities:
+You can launch NFC applications based on the NDEF message content using the [application control](./app-management/app-controls.md) functionalities:
```

Comment:
It looks like not ./app-management/app-controls.md, but **..**/app-management/app-controls.md.

Reply:
done